configmap/parse.go

```diff
@@ -144,6 +146,33 @@ func AsQuantity(key string, target **resource.Quantity) ParseFunc {
 	}
 }
 
+// AsNamespacedName parses the value at key as a types.NamespacedName into the target, if it exists
```

Comment:
Probably makes sense to comment that this works more like k8s NN rather than Knative NN, where we permit empty NS (e.g. the cache split one).

Reply:
done